### PR TITLE
TINY-4780: Added eslint with all failing rules switched off

### PR DESCRIPTION
**Pull request by tiny-james**
Removed old eslint configuration files.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
+{
+  "extends": "plugin:@tinymce/standard",
+  "parserOptions": {
+    "sourceType": "module",
+    "project": [
+      "./tsconfig.json",
+      "modules/*/tsconfig.json"
```
Comment (TheSpyder) [inline .eslintrc.json]: eslint doesn't support project references? interesting 🤔

Comment (tiny-james) [reply .eslintrc.json]: Unless something has changed recently:
https://github.com/typescript-eslint/typescript-eslint/issues/856

**Comment by tiny-james** [comment]
@lnewson I plan to go through most of the rules right after this PR in small groups so as to not overload the review requirements.

**Comment by lnewson** [comment]
> @lnewson I plan to go through most of the rules right after this PR in small groups so as to not overload the review requirements.

Oh, cool. Sounds like a plan, thanks @tiny-james! 👍

File: .eslintrc.json (hunk 1 above)
Comment (TheSpyder) [reply .eslintrc.json]: well that sucks, it seems like it's by design

>  in particular, the goal of project references is that when project a references project b, project a does not include the .ts files in project b.

**Pull request merged**